tiflash/tiflash-configuration.md

```diff
@@ -87,6 +87,8 @@ This section introduces the configuration parameters of TiFlash.
 
 - The path in which the TiFlash temporary files are stored.
 - By default, it is the first directory in [`path`](#path) or in [`storage.latest.dir`](#dir-1) appended with `"/tmp"`.
+- Starting from v9.0.0, it is recommended that you use the [`storage.temp`](#storagetemp-new-in-v900) configuration instead of `tmp_path`, as it supports capacity limits to control temporary file space usage.
```

Comment:
```suggestion
- Starting from v9.0.0, it is recommended that you use the [`storage.temp`](#storagetemp-new-in-v900) configuration instead of `tmp_path`, because it supports setting a capacity limit to control temporary file space usage.
```